gulpfile.js

```diff
@@ -534,6 +645,10 @@ function listReleaseTasks(done) {
         releaseTasks.push(function release_linux32_rpm(done){ return release_rpm('linux32', done) });
     }
 
+    if (platforms.indexOf('armv7') !== -1) {
+        releaseTasks.push(function release_armv7_zip(){ return release_zip('armv7') });
```

Comment:
Space before brace missing.

Reply:
If you're referring to `release_armv7_zip(){`, I'm consistent with the rest of the script.

Reply:
Actually you are not:

```
mikeller@pc:~/git/betaflight-configurator$ grep -e "function.*) {" gulpfile.js | wc -l
34
mikeller@pc:~/git/betaflight-configurator$ grep -e "function.*){" gulpfile.js | wc -l
12
```

And yes, the script is already inconsistent in this respect.

In general, I don't like the use of single line function definitions - they make maintenance of the code harder by increasing the probability of merge conflicts.

Reply:
In this segment of code they could as well be anonymous functions, but naming them helps a bit while debugging I guess.

I'd suggest that this block (arch-specific build functions) should be allowed to have one liners & no space after the function's declaration, in order to strive for an overall shorter line length.

If you want to keep the spaced style I'll fix the rest of the occurrences in the file, but this might somehow clutter the scope of this PR.

Reply:
We can leave them for now and clean this up in another pull request.

I do not quite get what advantage you see in 'overall shorter line length', to me the goal should be to be consistent for all function definitions.